pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1864,6 +1941,7 @@ type buildListenerOpts struct {
 	proxy             *model.Proxy
 	bind              string
 	port              int
+	tlsOpts           *networking.Server_TLSOptions
```

Comment:
lets call this userTlsOpts to clearly distinguish the peer authentication security policy from application-centric TLS certs

Reply:
ok